documentation/solution-architecture.md

```diff
@@ -0,0 +1,151 @@
+
+As for the ETSI TS 103 171 standard we see a need for a similar standard based on JSON and possibly one that better aligns with a data landscape that holds public keys in official distributed ledgers in the open apart from signed data shared privately. Though it is easy to create a XaDES export of any verifiable proof created from the waardepapieren solution, we do not see a direct need for this however for this project in it's current stage and also defer this topic to ongoing development of Discipl.
+
+By using NLX certificates for signing and information retrieval, the waardepapieren service will align with REGULATION (EU) No 910/2014 for a large extend, the rest of which is to be deferred to the ongoing development of Discipl like as with ETSI TS 103 171.
+
+The Waardepapieren solution should adhere to EN 301 549 / WCAG 2.0 though this is largely dependent on the integration of the solution in the user interfaces at the side of the issuer and not the solution itself. The project does involve the implementation of a reference validator app  / wallet however which must adhere to EN 301 549 / WCAG 2.0.
+
+As said, the solution also aims at a W3C verifiable credential compliant solution. This compliancy is deferred to the ongoing development of Discipl however. The solution adheres to the main mandatory concepts of this and related standards however.
+
+## Organisational layer
+
+The following stakeholders are identified:
+
+- issuer : automated (abundance) service available at location of issuer and/or through its website
+- clerk : employee that helps a client at location of issuer, services, identifies and authorizes the client manually
+- authorization service : automated service that identifies and authorizes client when client self services him/herself and accesses issuer through website
+- client : person in need of verifiable information required by a validator. It also becomes a holder of the resulting signed information.
+- validator : actor that needs to receive / verify information from client
+- administrator : actor that controls and monitors the automated issuer service
+
+An example of the essential conversation between real people as actors that would take place
+in the use cases the waardepapieren solution facilitates is given below:
+
+![](images/vcseq0.png)
+
+In essence this is the very proces that also takes place when retrieving a proof through the waardepapieren solution as can be seen below in the other verifiable claim sequence diagrams.
```

Comment:
proces -> process

I was kind of expecting the diagram below to have some relation to the diagram below. 

Is this the place to define "Verifiable Claims Sequence Diagrams"? This information is only used much later. Maybe we would want to move this out to a separate page and then link to it before the more intimidating usage later on?

Reply:
I guess we should include the definition of these types of diagrams to the main repository to contain a corresponding ISO 42010 viewport description and refer to that when talking about these kind of diagrams

Reply:
I removed the inline picture explaining this type of diagrams and added a link to it instead at the point where the type of diagram is first mentioned